### ISSUE TITLE
MSTKDNet_wo_gsm used instead of full model and GSM module disabled in training

### ISSUE THREAD

**Issue by alessiasaporita** [Quanato607/MST-KDNet#2]
Hi, thanks for releasing the code!
While trying to reproduce the paper, I noticed something that seems inconsistent with the method described.

1. The model is instantiated as:

`build_MSTKDNet_wo_gsm(inp_dim1=4, inp_dim2=4)`

which suggests that the GSM module is removed. However, in the paper the GSME (Global Style Matching Enhancement) is a key component of the proposed method. Additionally, the code imports:

`from .MSTKDNet_wo_gsm import MSTKDNet_wo_gsm`

but the corresponding file does not seem to be present in the repository.

2.  Although a GSM loss is defined in the codebase, it appears to be disabled during training.

Is the released training code using an ablation version of the model (without GSM)?
Is there an implementation of the full MST-KDNet with GSME available?

Any clarification would be very helpful.

Thanks!
